src/en/proposals/fairlysadpanda-revs-uplink.md

```diff
@@ -0,0 +1,160 @@
+
+_\*A 2.0 implementation of the Rev conversion mechanics should/must have a way of neutralizing the Head Rev conversion ability, allowing them to remain in the round._
+
+# The Uplink Terminal
+
+The **Centcomm Uplink Terminal** is a very tough computer console that lives somewhere on the ship. It's generally assumed it lives on the bridge, but it's actually mapper's choice where it goes. However, some general suggestions are:
+
+1. The location should be in an important room, as it represents the core security nexus for the entire station.
+2. The location should be somewhat defensible, or easily convertable to being so with some supplies to create barricades and walls.
+3. The location shouldn't be directly accessible by Security, as this makes attacking it and defending it a bit too easy for that department.
+
+Note that this sort of traditional attack/defend level design is not that normal in mapping, so it's OK if one station or another is more easy or more difficult to defend. As noted in [the rules section above](#the-rules), some stations might be found to to simply not have a good location to add an Uplink Terminal to.
+
+## The Fluff/The Lore
+
+The Uplink Terminal is the computer responsible for talking to CentComm. It's where CentComm messages come in to when sent, and where CentComm sends crew manifests, security information, and other important information to. The Terminal is responsible for setting up door access before the crew starts their shift. It also reports back metrics to CentComm like "what's the alert level", "where is the station at the moment" and "is the station in tiny bits".
+
+The existence of these terminals and the ability to transfer control of stations between security systems is due to a right-to-repair clause in Space Law to prevent old space stations from going to waste. Nanotrasen's used this law to repurpose such popular space stations it's found (floating adrift in deep space) as Barratry.
+
+During rounds, it's the fancy way of sending communications. It's like when a captain in a Star Trek episode needs to stand somewhere on the bridge to send a message to the crew to abandon ship or something.
+
+## The Terminal Is Robust
+
+The terminal is really hardy. It's an indepentent computer structure which is indepentently powered and has a high amount of structure damage resistance. It can't be broken into with tools or unanchored with tools. Essentially, it's as hardy as CentComm can reasonably make it. It can still be nuked, eaten by a singuloose, and so on.
+
+## (It is not infinitely robust)
+
+If the terminal is destroyed (except by nuke) then CentComm will send an automated annoyed message to the crew and send a replacement computer board to the automated trade station. This is absolutely a way that the Revolution could game the system, but considering that they'd need to do something very drastic to do so...
```

Comment:
Wouldn't this encourage revolutionaries to break the rules and singuloose in order to move the terminal to a more convenient location?

Reply:
"We overthrew Nanotrasen and only had to tesloose to do it" is pretty funny

Reply:
I'm pretty sure doing this is against the rules currently,  I believe revolutionaries aren't meant to be causing station-wide damage (You even say it yourself in the overview - the goal of the revs is to deliver the station, mostly intact, to the syndicate). 

I'm not against preventive measures to avoid metagaming from the heads, but fighting it with metagaming from the revs feels like a pitfall just ready to cause a ton of issues.

Reply:
This might not be completely fixable (limits in sandbox game == hard) but I get your point. Fundamentally any objective-based game mode has to handle "ok, so what happens if the objective has been rendered impossible".

The trick is making a countermeasure without causing a negative play experience via someone gaming the system.

I'm gonna try and organize a design discussion on this and see what comes out, as I'm capped out on ideas here.

Reply:
This might not be the best way to handle it, but can't we just make it work in the same way as other currently indestructible objects? (AKA nuke disk and nuke)

Reply:
That was my thought too. Indestructible seems like a simple solution

Reply:
Could make it surrounded in a bubble that bounces the singulo/tesla like the containment field does.